ru/news/_posts/2019-03-13-ruby-2-5-4-released.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: news_post
+title: "Вышел Ruby 2.5.4"
+author: "nagachika"
+translator: "lex111"
+date: 2019-03-13 11:30:00 +0000
+lang: ru
+---
+
+Вышел релиз Ruby 2.5.4.
+
+Этот релиз включает исправления ошибок и обновление безопасности RubyGems.
+Подробную информацию смотрите в [Многочисленные уязвимости в RubyGems](/en/news/2019/03/05/multiple-vulnerabilities-in-rubygems/)
```

Comment:
@lex111 maybe it makes sense to translate this one as well, just to have proper consistent links from scratch?

Reply:
@razum2um thank you for the review!

>  maybe it makes sense to translate this one as well, just to have proper consistent links from scratch?

No, I don't think so.